src/views/shared/exit/exit.view.jsx

```diff
@@ -286,18 +289,22 @@ function Exit({
           const pendingDelayedWithdrawal = pendingDelayedWithdraws?.find(
             (pendingDelayedWithdrawal) => pendingDelayedWithdrawal.id === exitId
           );
+          const timerWithdraw = timerWithdraws?.find(
+            (timerWithdraws) => timerWithdraws.id === exitId
+          );
 
-          if (pendingDelayedWithdrawal) {
-            const remainingTime = getDateString(pendingDelayedWithdrawal);
+          if (pendingDelayedWithdrawal || timerWithdraw) {
+            const withdraw = pendingDelayedWithdrawal || timerWithdraw;
```

Comment:
For better readability, I'd move this line up and test for `withdraw` in the if condition.